Library/Homebrew/cmd/test-bot.rb

```diff
@@ -209,19 +209,16 @@ def initialize(argument, options={})
       @added_formulae = []
       @modified_formula = []
       @steps = []
-      @tap = options[:tap]
+      @tap = options.fetch(:tap,  CoreFormulaRepository.instance)
```

Comment:
This feels a bit weird to me; it'd be nice to have a method that to give me that instance.